### ISSUE TITLE
[Accessibility] Enable ARIA reflection (non-idref) on Nightly

### ISSUE THREAD

**Issue by Rumyra** [mdn/content#26689]
## Acceptance Criteria

- [ ] The listed features are documented sufficiently on MDN
- [ ] BCD is updated
- [ ] Interactive example and data repos are updated if appropriate
- [ ] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [ ] Set bugs to `dev-doc-complete`
- [ ] Add entry to Firefox release notes if feature is enabled in release
- [ ] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Related Gecko bugs

https://bugzilla.mozilla.org/show_bug.cgi?id=1824980

## Other

- [ ] Check content open issues to see if any pertain to the subject matter. If there are any that can be closed because of the work, do so. If there are any that can be fixed relatively quickly because of the knowledge from completing this issue and you have time, feel free to go ahead and fix them.
- [ ] Check if glossary updates are required for the feature you're documenting - whether an existing term needs to be updated or a new term should be added.
- [ ] Check if BCD update means that content pages need to have experimental markup removed or deprecated markup added (front matter tags and macros).

## Related issues and pull requests

- [ ] https://github.com/mdn/content/pull/27117
- [ ] BCD?

**Comment by dipikabh** [comment]
Note for whoever takes this on: Related bug to also check out when working on this doc issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1572512

**Comment by bsmth** [comment]
Useful resources:

* https://github.com/WICG/aom/blob/gh-pages/explainer.md#reflecting-aria-attributes
* WPT: [Fx113, Fx115, Chrome stable, Safari stable](https://wpt.fyi/results/html/dom/aria-attribute-reflection.html?label=master&product=firefox%5Bstable%5D&product=firefox%5Bexperimental%5D&product=safari%5Bstable%5D&product=chrome%5Bstable%5D&aligned&q=aria)

**Comment by dipikabh** [comment]
Marking this as complete

**Comment by dipikabh** [comment]
Marking this as complete
